src/python/ensembl/io/genomio/gff3/process.py

```diff
@@ -549,7 +553,7 @@ def transcript_gene(self, ncrna: SeqFeature) -> SeqFeature:
     def cds_gene(self, cds: SeqFeature) -> SeqFeature:
         """Returns a gene created for a lone CDS."""
 
-        print("Put the lone CDS in gene-mRNA parent features")
+        logging.info("Put the lone CDS in gene-mRNA parent features")
```

Comment:
Do we want to add some more information about what CDS is it about?

Reply:
Yes; done